app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -1439,7 +1462,7 @@ public void onBackPressed() {
         }
         if (atHome) {
             if (backPressedOnce) {
-                finish();
+                finishAffinity();
```

Comment:
But, we can also set the intent flags for this activity as 
`Intent.FLAG_ACTIVITY_NEW_TASK`
and `Intent.FLAG_ACTIVITY_CLEAR_TASK`

It will do the same job as `finishAffinity()`

https://stackoverflow.com/questions/33497151/activity-finishaffinity-vs-intent-flag-activity-new-task-intent-flag-activit